packages/frontend/src/components/Game/GameBoard/Tile/index.jsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-const Tile = ({ tile, onFlip }) => {
+const Tile = ({ tile, onFlip, player }) => {
```

Comment:
If the `currentPlayer` value in the context is just the player's index, it could be changed to be the entire player object of the active player. Then the `player` value could be accessed from the context instead of being passed as a prop.